modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/service/taglib/BaseDynamicInclude.java

```diff
@@ -103,9 +101,8 @@ public void include(
 
 		try {
 			List<CET> cets = cetManager.getCETs(
-				themeDisplay.getCompanyId(), null,
-				ClientExtensionEntryConstants.TYPE_GLOBAL_JS,
-				Pagination.of(QueryUtil.ALL_POS, QueryUtil.ALL_POS), null);
+				themeDisplay.getCompanyId(),
+				ClientExtensionEntryConstants.TYPE_GLOBAL_JS);
```

Comment:
Hey @thiago-buarqque Isn't this the case where we want to return the instance scoped CET's too? So it should be calling the longer method?

Reply:
Never mind, I misunderstood what the default was.